openshift/tests-extension/test/webhooks.go

```diff
@@ -167,6 +168,64 @@ var _ = Describe("[sig-olmv1][OCPFeatureGate:NewOLMWebhookProviderOpenshiftServi
 			}))
 		})
 
+		It("should be tolerant to openshift-service-ca certificate rotation", func(ctx SpecContext) {
+			By("deleting the openshift-service-ca signing-key secret")
+			signingKeySecret := &corev1.Secret{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      openshiftServiceCASigningKeySecretName,
+					Namespace: openshiftServiceCANamespace,
+				},
+			}
+			err := k8sClient.Delete(ctx, signingKeySecret, client.PropagationPolicy(metav1.DeletePropagationBackground))
+			Expect(client.IgnoreNotFound(err)).ToNot(HaveOccurred())
+
+			By("waiting for the webhook operator's service certificate secret to be recreated and populated")
```

Comment:
See that now we wait and ensure first

Reply:
Do we need all of this extra code to check if the secret is recreated though? The next `Eventually` block will fail anyway if the secret isn't recreated - and we'll know why when we dump the info on failure (from line 210)

Reply:
Hi @perdasilva thank you for looking on this

I addressed your suggestions for we either more conservative within, but now we need wait to pass in the tests here + a new agreegate 
So 

/hold

Until we have all in place again

Reply:
/hold